pkg/policy/policy.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2023 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policy
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"os"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	"knative.dev/pkg/apis"
+)
+
+type Verification struct {
+	// NoMatchPolicy specifies the behavior when a base image doesn't match any
+	// of the listed policies.  It allows the values: allow, deny, and warn.
+	NoMatchPolicy string `yaml:"no-match-policy,omitempty"`
+
+	// Policies specifies a collection of policies to use to cover the base
```

Comment:
Yeah the "base image" thing was because I documented some of this before copying it over from `ko`, this is definitely stale.  The rest is a little confusing because I renamed this field twice.  I'll update it in the next push.